tvb_framework/tvb/interfaces/rest/bids_monitoring/launch_bids_monitor.py

```diff
@@ -0,0 +1,46 @@
+import sys
+from tvb.interfaces.rest.bids_monitoring.bids_data_builder import BIDSDataBuilder
+from tvb.interfaces.rest.bids_monitoring.bids_dir_monitor import BIDSDirWatcher
+from tvb.adapters.uploaders.bids_importer import BIDSImporter
+
+
+BIDS_UPLOAD_CONTENT = BIDSImporter.NET_TOKEN
+BIDS_DIR = "C:/Users/upadh/Documents/GitHub/tvb-root/tvb_framework/tvb/interfaces/rest/client/examples/BIDS_DEMO_DATSET - Copy"
```

Comment:
This hard-coded path might be wrong to keep here ?

Reply:
yes, removing it